gbpservice/nfp/service_vendor_agents/vyos/agent/DEBIAN/control

```diff
@@ -0,0 +1,8 @@
+Package: vyos
+Source: vyos
+Version: 2.5
+Architecture: all
+Maintainer: One Convergence
+Section: devel
+Priority: optional
+Description: vyos package
```

Comment:
Address what @pkharat mentioned as decision of where to submit is independent of this.